package.json

```diff
@@ -9,7 +9,7 @@
   },
   "scripts": {
     "dev": "cp ./template.html ./index.html && vite",
-    "build": "pnpm run build:set-last-commit-and-tag && pnpm run build:generate-entry-points && tsc && vite build",
+    "build": "NODE_OPTIONS='--max_old_space_size=8192' pnpm run build:set-last-commit-and-tag && pnpm run build:generate-entry-points && tsc && vite build",
```

Comment:
our builds are starting to OOM. vercel points [out 3 primary suspects](https://vercel.com/guides/troubleshooting-sigkill-out-of-memory-errors#why-builds-fail-due-to-memory-issues):
- large number of dependencies
- large data handling
- inefficient code

we haven't added any data handling and none of the code here is really has the opportunity to very inefficient in terms of build size.

so the primary theory is that our dependencies list has grown big enough where we need to up our node limits. if node crashes via OOM it can cause the vercel container to 137 exit, so we're following [vercel's recommendation to increase the node heap size](https://vercel.com/guides/troubleshooting-sigkill-out-of-memory-errors#4.-increase-memory-allocation-for-node)